### ISSUE TITLE
Unable to see sparsebundle

### ISSUE THREAD

**Issue by crankygerman** [fman-community/fman#579]
Hi,

I'm unable to see a sparsebundle created by TimeMachine stored on an external drive in macOS Mojave. I tried to "Toggle hidden files" but it made no difference. How can I make it visible?

Best, Cranky

![image](https://user-images.githubusercontent.com/45127838/48850918-0772fe00-edaa-11e8-8f86-12de9af90968.png)

**Comment by mherrmann** [comment]
Can you see it in Finder?

**Comment by crankygerman** [comment]
Yes, that is what the attached screenshot is intended show. Finder is at the bottom and fman at the top.

**Comment by mherrmann** [comment]
Oh, I see, sorry. I thought the window at the bottom was TimeMachine, not Finder.

The above shows a volume, at `/Volumes/XYZ`. What do you get when you do

    ls -al /Volumes/XYZ/Backups.backupdb

in a Terminal?

**Comment by crankygerman** [comment]
"Operation not permitted"

**Comment by mherrmann** [comment]
I suspect that's the reason then: fman doesn't have permissions to read from the location. It should give at least give a message though, so you know what's going on.

**Comment by crankygerman** [comment]
Thank you, seems to be a reasonable explanation. I'll try to figure out how to give fman the appropriate permissions. At first glance the permissions set on Backups.backupdb and the contained subfolders appear to be identical.
